api/Document.json

```
@@ -265,26 +279,64 @@
             "ie": {
               "version_added": "4"
             },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": true
-            },
+            "opera": [
+              {
+                "version_added": "51"
+              },
+              {
+                "version_added": "15",
```

```suggestion
                "version_added": "≤12.1"
```

Attributes aren't on the prototype in Presto, but you can tell that `document.all` was only supported for `HTMLDocument` by creating different documents using https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/createDocument

As a future improvement to the collector it would be good to test all `Document` members using both an `HTMLDocument` and an `XMLDocument`, and also a plain `Document` if the `new Document()` constructor is supported.